pkg/errors/errorType.go

```diff
@@ -0,0 +1,62 @@
+package errors
```

Comment:
Will be problems with the file name? Could we move it to errros.go?

Reply:
we don't need more the built-in errors package. Our package overwrites the error. But we can use both with the same name without problems